sources/HUBViewControllerImplementation.m

```diff
@@ -865,6 +866,9 @@ - (void)createCollectionViewIfNeeded
     collectionView.delegate = self;
     
     [self.view insertSubview:collectionView atIndex:0];
```

Comment:
You're now already doing this in `HUBContainerView`. So remove this to avoid doing this twice.